src/python/export_module_manager.cpp

```diff
@@ -19,9 +19,21 @@
 #include <pluginplay/any/any.hpp>
 #include <pluginplay/module_manager.hpp>
 #include <pluginplay/python/python_wrapper.hpp>
+#include <pybind11/stl.h>
 
 namespace pluginplay {
 
+void export_module_pair(py_module_reference m) {
+    py_class_type<std::pair<const std::string, std::shared_ptr<Module>>>(
+      m, "ModulePair")
+      .def(pybind11::init<>())
+      .def_readonly(
+        "first", &std::pair<const std::string, std::shared_ptr<Module>>::first)
+      .def_readonly(
+        "second",
+        &std::pair<const std::string, std::shared_ptr<Module>>::second);
+}
```

Comment:
```suggestion
```
Pybind11 should auto convert `std::pair` to a Python tuple as long as the types in the pair are exposed (which `std::string` is exposed by Pybind11 and we should have already exposed `Module`). Generally speaking you can't expose `const`, `&`, and `std::shared_ptr` to Python because it has no sense of those concepts.

Reply:
Thanks, I forgot to delete this one.